### PR TITLE
Issues-226: Add docs for custom Cucumber reporters

### PR DESCRIPTION
**Pull request by flashbackzoo**
Fixes https://github.com/mucsi96/nightwatch-api/issues/226

### CODE REVIEW

File: packages/website/src/data/cucumber/intro/custom-reporters.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+ For details on available options, see the
+[Cucumber HTML Reporter documentation](https://github.com/gkushang/cucumber-html-reporter#usage).
+
+ ### Step 3 - Update your npm scripts
+
+ You may have noticed in the first step we installed `npm-run-all`. This lets us run a sequence of npm scripts, while
+avoiding cross platform issues operators such as `&&` introduce. Another handy feature is the ability to continue the
+sequence when a previous script fails. This is important because we want to generate a test report even when our tests
+fail, and cause Nightwatch to throw an error.
```
Comment (spnraju) [inline packages/website/src/data/cucumber/intro/custom-reporters.md]: Should we mention somewhere that we can also run this as a separate process to generate the report?

Comment (spnraju) [reply packages/website/src/data/cucumber/intro/custom-reporters.md]: I think we can have it this way for now. Probably we can extend it at a later point of time. Thank you for the contribution @flashbackzoo .

**Comment by mucsi96** [comment]
@flashbackzoo Please review: https://github.com/mucsi96/nightwatch-api/pull/304

**Comment by mucsi96** [comment]
I managed to do the report generation in the same process using setTimeout

**Pull request merged**